sematic/api/endpoints/resolutions.py

```diff
@@ -0,0 +1,90 @@
+"""
+Module keeping all /api/v*/runs/* API endpoints.
+"""
+
+# Standard Library
+from http import HTTPStatus
+from typing import Optional
+
+# Third-party
+import flask
+from sqlalchemy.orm.exc import NoResultFound
+
+# Sematic
+from sematic.api.app import sematic_api
+from sematic.api.endpoints.auth import authenticate
+from sematic.api.endpoints.request_parameters import jsonify_error
+from sematic.db.models.resolution import Resolution
+from sematic.db.models.user import User
+from sematic.db.queries import get_resolution, get_run, save_resolution
+
+
+@sematic_api.route("/api/v1/resolutions/<resolution_id>", methods=["GET"])
+@authenticate
+def get_resolution_endpoint(user: Optional[User], resolution_id: str) -> flask.Response:
+    try:
+        resolution = get_resolution(resolution_id)
+    except NoResultFound:
+        return jsonify_error(
+            "No resolutions with id {}".format(repr(resolution_id)),
+            HTTPStatus.NOT_FOUND,
+        )
+
+    payload = dict(
+        content=resolution.to_json_encodable(),
+    )
+
+    return flask.jsonify(payload)
+
+
+@sematic_api.route("/api/v1/resolutions/<resolution_id>", methods=["PUT"])
+@authenticate
+def put_resolution_endpoint(user: Optional[User], resolution_id: str) -> flask.Response:
```

Comment:
Do we even need to update resolutions at this time? If we remove the state machine, what do we not know at creation time?